homeassistant/components/lg_soundbar/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Service is already configured",
+            "existing_instance_updated": "Updated existing configuration."
+        },
+        "error": {
+            "cannot_connect": "Failed to connect"
+        },
+        "step": {
+            "details": {
```

Comment:
This is what I'm talking about. Shouldn't this be `user`? Defined at line 70 and 78 in `config_flow.py`. When I'm changing it to `user`, I'm getting the strings I'm talking about. Compare this image with the other I posted.
<img width="394" alt="image" src="https://user-images.githubusercontent.com/270292/174161465-6eaa3179-08e1-4407-adc1-55eac59dcff7.png">

```suggestion
            "user": {
```

Reply:
nice find, missed that when i renamed the config_flow